Test fails due to missing CLI docs destination directory

**Description**
If the directory `docs/cli` does not exist (e.g. new clone of repository), one test fails as it tries to list files for a directory which does not exist, leading to `no such file or directory`.

**Reproduction Steps**
Steps to reproduce the behavior:
1. Pull clean copy of repository.
1. Change directory to repository root.
1. Run `make test`.
1. At least one test fails.

**What happened**
The following output is shown:

```
go test -race -coverprofile=c.out ./...
go tool cover -html=c.out -o=coverage.html
        github.com/miniscruff/changie           coverage: 0.0% of statements
        github.com/miniscruff/changie/then              coverage: 0.0% of statements
✔ Kind … Changed
✔ Body … older
✔ Kind … Changed
✔ Body … newer
Error: invalid argument "blah-blah-blah" for "changie next"
Usage:
  changie next major|minor|patch|auto [flags]

Flags:
  -h, --help                 help for next
  -i, --include strings      Include extra directories to search for change files, relative to change directory
  -m, --metadata strings     Metadata values to append to version
  -p, --prerelease strings   Prerelease values to append to version
  -j, --project string       Specify which project we are interested in

Error: invalid argument "v1.2.3" for "changie next"
Usage:
  changie next major|minor|patch|auto [flags]

Flags:
  -h, --help                 help for next
  -i, --include strings      Include extra directories to search for change files, relative to change directory
  -m, --metadata strings     Metadata values to append to version
  -p, --prerelease strings   Prerelease values to append to version
  -j, --project string       Specify which project we are interested in

--- FAIL: TestCanGetFiles (0.00s)
    gen_test.go:45: expected 'open docs/cli: no such file or directory' to be nil
                                    ✔ Kind … removed
✔ Body … a message with testcontent
                                    ✔ Kind … removed
✔ Body … a message
                                    ✔ Kind … removed
✖ Body … a message
                                    ✔ Kind … removed
✔ Body … a message
✔ Body … another body
                                    ✔ Component … test/component
                                    ✔ Kind … removed
✔ Body … a message
FAIL
coverage: 82.2% of statements
FAIL    github.com/miniscruff/changie/cmd       3.463s
ok      github.com/miniscruff/changie/core      3.471s  coverage: 97.6% of statements
FAIL
```

**Expected behavior**
All tests should succeed.

**Additional context**
Nothing.

--- a/cmd/gen_test.go
+++ b/cmd/gen_test.go
@@ -41,6 +41,10 @@ func TestCanGetFiles(t *testing.T) {
 	// ignore any bad error trying to remove the file, it may not exist and that is ok
 	_ = os.Remove(configDocsPath)
 
+	// create cli docs directory if it does not already exist
+	err = os.MkdirAll(cliDocsPath, os.ModePerm)
+	then.Nil(t, err)
+
 	cliDocsFiles, err := os.ReadDir(cliDocsPath)
 	then.Nil(t, err)
 

